builder/src/build/patches/investigated/update-webpack-chunks-file/get-file-content-with-updated-webpack-require-f-code.ts

```diff
@@ -0,0 +1,74 @@
+import * as ts from "ts-morph";
+
+export async function getFileContentWithUpdatedWebpackFRequireCode(
```

Comment:
Could you add some comments explaining how this all works?

Reply:
comment added 🙂